cmd/pkger/cmds/pack.go

```diff
@@ -45,15 +45,12 @@ func (e *packCmd) Exec(args []string) error {
 		return err
 	}
 
-	fp := filepath.Join(info.Dir, e.out, outName)
-	os.RemoveAll(fp)
-
 	decls, err := parser.Parse(info, e.include...)
 	if err != nil {
 		return err
 	}
 
-	if err := Package(info, fp, decls); err != nil {
+	if err := Package(info, e.out, decls); err != nil {
```

Comment:
I don't believe `fp` and `e.out` are equal, so I would be concerned that this would change behavior unexpectedly.

Reply:
Yes, `fp` and `e.out` aren't the same.
I did this to reduce the scope of `fp`, because it's only use was to call `os.RemoveAll()` and call `Package()` which also calls `os.RemoveAll()`...